Testing/functional/tests/cypress/integration/ui/covid19/publicVaccineCard.js

```diff
@@ -217,16 +217,19 @@ describe("Public Vaccine Card Form", () => {
 
 describe("Public Vaccine Card Downloads", () => {
     beforeEach(() => {
-        cy.configureSettings(
```

Comment:
Why are there ui changes?  The feature branch is already changed.

https://github.com/bcgov/healthgateway/blob/features/13691/Testing/functional/tests/cypress/integration/ui/pages/publicRoute.js